nixio/pycore/h5dataset.py

```diff
@@ -43,6 +43,8 @@ def write_data(self, data, count=None, offset=None):
             self.dataset[:] = data
 
     def read_data(self, data, count=None, offset=None):
+        if 0 in self.dataset.shape or len(self.dataset.shape) == 0:
+            return
```

Comment:
Does this work? Do we need to switch the checks? In case dataset has no shape does the first check pass?
Just wondering if something like would be safer:
```
if len(self.dataset.shape) < 1 and 0 in self.dataset.shape:
         return
```